pkg/operator/ceohelpers/bootstrap_test.go

```diff
@@ -333,15 +363,19 @@ func Test_CheckSafeToScaleCluster(t *testing.T) {
 				}
 			}
 			fakeInfraStructure := configv1listers.NewInfrastructureLister(fakeInfraIndexer)
+			fakeEtcdClient, err := etcdcli.NewFakeEtcdClient(test.etcdMembers)
+			if err != nil {
+				t.Fatal(err)
+			}
 
-			actualErr := CheckSafeToScaleCluster(fakeConfigMapLister, fakeStaticPodClient, fakeNamespaceMapLister, fakeInfraStructure)
+			actualErr := CheckSafeToScaleCluster(
+				fakeConfigMapLister,
+				fakeStaticPodClient,
+				fakeNamespaceMapLister,
+				fakeInfraStructure,
+				fakeEtcdClient)
 
-			if test.expectError != nil && actualErr == nil {
-				t.Errorf("expected error=%v, got %v", test.expectError, actualErr)
-			}
-			if test.expectError == nil && actualErr != nil {
-				t.Errorf("expected error=%v, got %v", test.expectError, actualErr)
-			}
+			assert.Equal(t, test.expectError, actualErr)
```

Comment:
Nice, I was confused at the behavior before this until I realized we were never checking the error.